### PR TITLE
fix neume image path -clean

### PR DESCRIPTION
**Pull request by yinanazhou**
Corrected neumes image path, and uploaded missing images in the `neume `folder (cleaned). [Notice](https://github.com/music-encoding/music-encoding/discussions/829): the images will only show up when using **command lines** to build HTML files. To make the **Oxygen** method work, the `image `folder needs to be pasted to `music-encoding\source\web` manually.

### CODE REVIEW

File: source/docs/06-neumes.xml [hunk 1]
```diff
@@ -125,9 +125,17 @@
...
                      <row>
-                        <cell>!<ref target="/guidelines/images/v4/modules/neumes/NEUME-con.png">Connected</ref>
+                        <cell>
+                          <figure>
+                            <head/>
+                            <graphic url="../images/modules/neumes/NEUME-con.png"/>
+                          </figure>
                         </cell>
-                        <cell>!<ref target="/guidelines/images/v4/modules/neumes/NEUME-non-con.png">Non-connected</ref>
+                        <cell>
```
Comment (ahankinson) [inline source/docs/06-neumes.xml:130]: This change seems to lose the 'Connected' and 'Non-connected' labels?

File: source/docs/06-neumes.xml [hunk 2]
```diff
@@ -996,7 +1143,11 @@
                         <cell>
                            <hi rend="italic">QUILISMA - Staff notation</hi>
                         </cell>
-                        <cell>!<ref target="/guidelines/images/v4/modules/neumes/quilisma.png">Quilisma</ref>
+                        <cell>
+                          <figure>
+                            <head/>
```
Comment (ahankinson) [inline source/docs/06-neumes.xml]: Is the empty `head` tag required?

**Review by ahankinson — CHANGES_REQUESTED** [review]

**Comment by yinanazhou** [comment]
Thanks for pointing that out. The labels can be put inside the `<head>` message. The resulting figure caption is going to be `Figure XX. Connected` if we add the label in the empty `<head>` message. We can add the labels if necessary.

**Comment by ahankinson** [comment]
They seem necessary?

**Comment by yinanazhou** [comment]
OK! We will add them right now!

**Comment by ahankinson** [comment]
Great! Thank you.

Just to make it easier to review without needing to download and run the code locally, could you please provide a screenshot of what it looks like now? Just one or two examples showing how the label and image look together would be sufficient.

**Comment by yinanazhou** [comment]
Hi, we just built it using command lines, and it looks like this now:
connected
nc

**Comment by ahankinson** [comment]
OK -- do you know where those funny `:-------:` things come from?

**Comment by ahankinson** [comment]
Ah, looking at it more, it seems that the 'connected' and 'non-connected' labels are in some sort of table? And the `:-----:` is maybe a hangover from the markdown conversion? @bwbohl @kepper do you know anything about this?

Should we clean those up now, or should we open another issue for it?

**Comment by yinanazhou** [comment]
They are in the [original version](https://github.com/music-encoding/music-encoding/blob/develop/source/docs/06-neumes.xml). We did not change it.
Example (line 123-126): 
```
<row>
      <cell>:---------:</cell>
      <cell>:---------:</cell>
</row>
```

**Comment by kepper** [comment]
Yes, if I remember correctly, those are artifacts from how figures and captions were organized "creatively" in Markdown. Would be perfect to clean that up while you're at it now… (pretty please ;-) From my perspective, those examples don't need to be written as table, and using a table for layout purposes is generally questionable, isn't it?

**Comment by ahankinson** [comment]
@yinanazhou would you be able to make the changes?

**Comment by musicEnfanthen** [comment]
The hyphens within colons mark a "center-aligned" table cell in markdown: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#formatting-content-within-your-table
All those [`<rows>` artifacts](https://github.com/music-encoding/music-encoding/pull/832#issuecomment-866966889) could be safely removed.

**Comment by yinanazhou** [comment]
@ahankinson yes, sure. We will do it now.

**Comment by yinanazhou** [comment]
Hi @ahankinson and @martha-thomae, this is how it looks like now:
image

**Comment by ahankinson** [comment]
Looks much better! Thank you.

@martha-thomae @kepper would you be willing to be the second approver on this PR? If so, feel free to merge if you're happy.

**Review by ahankinson — APPROVED** [review]

**Review by bwbohl — APPROVED** [review]

**Pull request merged**